yarn.lock

```diff
@@ -7410,7 +7410,7 @@ karma-source-map-support@1.4.0:
   dependencies:
     source-map-support "^0.5.5"
 
-katex@0.10.2, katex@^0.10.0:
+katex@^0.10.0:
```

Comment:
This could have been the main issue, having two versions of katex (one as a dependency of showdown-katex, one installed manually).